docs/concepts/controller-manager.md

```diff
@@ -336,7 +336,7 @@ If the shoot cluster is older than the configured lifetime, then it gets deleted
 It maintains the expiration time of the `Shoot` in the value of the `shoot.gardener.cloud/expiration-timestamp` annotation.
 This annotation might be overridden, however only by at most twice the value of the `.spec.clusterLifetimeDays`.
 
-#### ["Reference" Reconciler](../../pkg/controllermanager/controller/shoot/reference)
+#### ["Shoot-Reference" Reconciler](../../pkg/controllermanager/controller/shoot/reference)
```

Comment:
Why this change? This is already scoped to "shoot" because it's in the "shoot controller" section?

Reply:
You can't add controllers with the same name twice to the same manager (validation error will happen), so the controllers where actually renamed to `Shoot-Reference`, `Seed-Reference` and `Garden-Reference`.

Reply:
That's fine - but in the docs, we don't need to reflect these specifics necessarily.